apps/core/components/searchForm/PlacePicker/PlacePickerRenderer.js

```diff
@@ -5,22 +5,51 @@ import { QueryRenderer, graphql } from '@kiwicom/margarita-relay';
 
 import type { PlacePickerRendererQueryResponse } from './__generated__/PlacePickerRendererQuery.graphql';
 import PlacePickerContent from './PlacePickerContent';
+import {
+  MODAL_TYPE,
+  type modalType,
```

Comment:
I think it should be `ModalType` with uppercase on the first letter

Reply:
Changed!